dyson/expressions/fci.py

```diff
@@ -51,7 +51,7 @@ def __init__(
                 self._nelec = nelec
                 self._nmo = h1e.shape[0]
 
-            if e_ci is None:
+            if c_ci is None:
```

Comment:
should we do `e_ci is None` or `c_ci is None` here? I originally did `e_ci is None` because it's only needed for the chempot, whereas `c_ci` is always needed. thoughts?

Reply:
My thinking is that the code can calculate the moments without the energy but not without the CI vector. But ideally, both should be passed in.

Reply:
yeah ok i'm happy with that